src/shared/components/NavigationMenu/NavigationMenu.types.ts

```diff
@@ -3,6 +3,11 @@ import { ReactNode } from 'react';
 export type NavigationItem = {
   name?: string;
   label: string;
+  useModal?: boolean;
+  modalTitle?: string;
+  modalTitleParams?: {
```

Comment:
It appears that these props are not utilized elsewhere. Should we consider removing them?